core/src/auth/auth.controller.ts

```diff
@@ -39,4 +43,32 @@ export class AuthController {
     res.send();
     return;
   }
+
+  @Get('auth/createDefaultRoles')
+  async defaultRoles() {
+    // Give superuser access to everything
+    (Object.keys(Resource) as Array<keyof typeof Resource>).map(
+      async (resource) => {
+        (Object.keys(ResourceAction) as Array<keyof typeof ResourceAction>).map(
+          async (action) => {
+            this.addPermTryCatch('superuser', resource, action + ':any');
+          },
+        );
+      },
+    );
+
+    // Make Nigel a superuser
+    await this.authZService.addRoleForUser(
+      'ce5a3b01-f81c-4391-807c-4be84336ab19',
```

Comment:
I think we have a configuration for this (also hard-coded guid needs to be removed)

https://github.com/SprocketBot/sprocket/blob/v2/config.yaml#L35-L37

Reply:
Fixed, now loops over those names in the config.